woocommerce-gateway-ebanx.php

```diff
@@ -124,18 +125,8 @@ private function __construct()
 			add_action('save_post', array($this, 'ebanx_metabox_payment_link_save'));
 
 			/**
-			 * My account
+			 * My account card section has been removed as WooCommerce already provide card storage feature.
```

Comment:
Does WooCommerce provide this or WooCommerce Subscription provides this?

Reply:
WooCommerce. [Here's the reference](https://github.com/woocommerce/woocommerce/wiki/Payment-Token-API#step-0-extending-the-correct-gateway-base)